store/src/java/com/zimbra/qa/unittest/TestZClient.java

```diff
@@ -29,10 +33,10 @@
 import java.util.List;
 import java.util.Set;
 
-import junit.framework.Assert;
-import junit.framework.TestCase;
-
 import org.apache.commons.io.IOUtils;
+import org.junit.After;
```

Comment:
This seems like an unnecessary change that causes a lot more non-functional changes throughout the rest of the file. I don't think it achieves anything other then making the diff very big.